control-plane-operator/controllers/hostedcontrolplane/v2/assets/kube-scheduler/deployment.yaml

```diff
@@ -21,6 +21,8 @@ spec:
       - args:
         - --config=/etc/kubernetes/config/config.json
         - --cert-dir=/var/run/kubernetes
+        - --tls-cert-file=/etc/kubernetes/certs/server/tls.crt
+        - --tls-private-key-file=/etc/kubernetes/certs/server/tls.key
         - --secure-port=10259
         - --authentication-kubeconfig=/etc/kubernetes/kubeconfig/kubeconfig
         - --authorization-kubeconfig=/etc/kubernetes/kubeconfig/kubeconfig
```

Comment:
$ curl --key /etc/kubernetes/certs/server/tls.key --cert /etc/kubernetes/certs/server/tls.crt --cacert /etc/kubernetes/certs/serve/ca.crt https://kube-scheduler.clusters-abc.svc.cluster.local:10259/metrics/resources
{
  "kind": "Status",
  "apiVersion": "v1",
  "metadata": {},
  "status": "Failure",
  "message": "Unauthorized",
  "reason": "Unauthorized",
  "code": 401
}

seems we're also missing authorization here